### ISSUE TITLE
Ivy regression with ng-bootstrap: dropdown not located at the right place first time it's clicked

### ISSUE THREAD

**Issue by jnizet** [angular/angular#30166]
# 🐞 bug report

### Affected Package
Unknown

### Is this a regression?
Yes. It works fine with 8.0.0-rc.1 without Ivy, and also with 7.2.14

### Description
The first time an ng-bootstrap dropdown is clicked, the dropdown appears shifted to the right. 

## 🔬 Minimal Reproduction
Since it's an Ivy bug, I can't provide a stackblitz URL. But the bug can be reproduced by cloning https://github.com/jnizet/valuechangesivy, and following the instructions in the README (first bug).

## 🔥 Exception or Error
NA

## 🌍  Your Environment

**Angular Version:**
<pre><code>
Angular CLI: 8.0.0-rc.1
Node: 10.15.1
OS: darwin x64
Angular: 8.0.0-rc.1
... animations, cli, common, compiler, compiler-cli, core, forms
... language-service, platform-browser, platform-browser-dynamic
... router

Package                           Version
-----------------------------------------------------------
@angular-devkit/architect         0.800.0-rc.1
@angular-devkit/build-angular     0.800.0-rc.1
@angular-devkit/build-optimizer   0.800.0-rc.1
@angular-devkit/build-webpack     0.800.0-rc.1
@angular-devkit/core              8.0.0-rc.1
@angular-devkit/schematics        8.0.0-rc.1
@ngtools/webpack                  8.0.0-rc.1
@schematics/angular               8.0.0-rc.1
@schematics/update                0.800.0-rc.1
rxjs                              6.5.1
typescript                        3.4.5
webpack                           4.30.0
</code></pre>

**Anything else relevant?**

No. The bug happens on Chrome and Firefox.

**Comment by pkozlowski-opensource** [comment]
@jnizet could we take https://github.com/ng-bootstrap/ng-bootstrap specific issues to https://github.com/ng-bootstrap/ng-bootstrap firstly? As we speak @marclaval is running all ng-bootstrap tests with ivy and there are few issues in ivy that were discovered in the process. I don't think there is much point in opening issues in the Angular repo just yes, as we are still doing first pass on ng-bootstrap with ivy.

What would be super-helpful, though, is to move this error (and other findings based on your investigations) to https://github.com/ng-bootstrap/ng-bootstrap. What would be even cooler is having help narrowing down the issue so it can be reproduced in a simple unit test without ng-bootstrap involved.

tl;dr; could you please either move this issue to https://github.com/ng-bootstrap/ng-bootstrap or help investigating the root cause so we can have a simple unit test? This would speed up things immensely!

**Comment by jnizet** [comment]
Hi @pkozlowski-opensource 

I'll file the issue on ng-bootstrap (and try to investigate the issue if I find the time). 
Since Ivy is sold as fully backward-compatible, I thought filing it here was the good place: it should work without changes, shouldn't it?

**Comment by pkozlowski-opensource** [comment]
> Since Ivy is sold as fully backward-compatible, I thought filing it here was the good place: it should work without changes, shouldn't it?

Sure. Just saying that we've got more context on ng-bootstrap side to investigate / narrow it down.
